### PR TITLE
minor changes for readability

### PR DESCRIPTION
**Pull request by AndrewSerra**
## Proposed changes

Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request.
If it fixes a bug or resolves a feature request, be sure to link to that issue.

## Types of changes

What types of changes does your code introduce to static food data?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] refactor
- [ ] documentation
- [ ] other

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

Are tests included?
- [ ] yes (*bugfixes and features will not be merged without tests*)
- [ ] no

Breaking Changes?
- [ ] yes (*breaking changes will not be merged unless absolutely necessary*)
- [ ] no

List any relevant issue numbers:

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by AndrewSerra** [comment]
Of course, doing it now

**Comment by AndrewSerra** [comment]
A quick question, are the todos generated by the bot? because they were not in the document. The only todo I have is in dash.test.js

**Comment by atherdon** [comment]
I add those todos - bot is just triggering an error

>

**Comment by atherdon** [comment]
why?

>

**Comment by AndrewSerra** [comment]
I did not know you could add it and was curious. For the 'favs' key, I did not quite understand what you meant

**Comment by atherdon** [comment]
everytyhing looks good except this error:
```
  /home/travis/build/GroceriStar/sd/tests/favorites.test.js:6:9: Parsing error: Identifier 'favorites' has already been declared
```

about your question - for now, you can just pass at `favs` and array with few generated ids and it'll work at this current stage

**Comment by atherdon** [comment]
please fix that error and i'll merge your PR

**Comment by atherdon** [comment]
about your question - actually pull requests at github is like a branch. so yeah, as admin i can checkout not your repository but branch that was created with your latest changes. it's very helpful when i need not only to read your code but also see how it works with other parts of module as well

**Comment by atherdon** [comment]
should i merge this or wait for alother commits from you?

File: tests/favorites.test.js [hunk 1]
```diff
@@ -20,18 +24,26 @@ const schema = {
...
-const example = { 'ingredient_id': '4aacafd5-6106-11e9-9ffc-a3e52b2d927f',
-  'user_id': '4aacafd2-6106-11e9-9ffc-a3e52b2d927f',
-  'favs': 'desc for department1',
-  'grocery_id': '4aac88cf-6106-11e9-9ffc-a3e52b2d927f'
+
+const testId = __generateId();
+const favoriteIngredients = generateArrWithId(ingredients, 'ingredient_id');
+
+const example = {
+  'ingredient_id': testId,
```
Comment (atherdon) [inline tests/favorites.test.js]: all of this ids are related to different entities - they cannot be the same varialbe. this should be re done

**Comment by AndrewSerra** [comment]
I'm done with it the unsuccessful check is about two parts being similar but I don't think that has an effect. You can merge it.

File: tests/favorites.test.js (hunk 1 above)
Comment (AndrewSerra) [reply tests/favorites.test.js]: sure, the template before had the same id for all of them that's why I kept it like this. changing them now

**Comment by atherdon** [comment]
got it - please fix then issue with same ids and i'll merge changes

**Comment by AndrewSerra** [comment]
Sorry I wasn't totally sure so I went back and checked the first version and they are indeed different, I changed them already so they are different

**Comment by atherdon** [comment]
i'm talking about this: https://github.com/GroceriStar/sd/pull/482#pullrequestreview-246584360

**Comment by AndrewSerra** [comment]
Oh, got it.

**Comment by AndrewSerra** [comment]
So the favoriteIngredients variable is an array of generated ids?

**Comment by atherdon** [comment]
yes

**Comment by atherdon** [comment]
always show code please
because its hard

**Comment by atherdon** [comment]
#423

**Comment by atherdon** [comment]
**/home/travis/build/GroceriStar/sd/tests/favorites.test.js:21:4: Parsing error: Unexpected token <<
error Command failed with exit code 1.``**

**Comment by AndrewSerra** [comment]
I do not know what the travis-ci error is and could not understand

**Comment by AndrewSerra** [comment]
the tests worked on my device and my repo is all up to date

**Comment by atherdon** [comment]
this is the latest error log from travis ci:
```
 /home/travis/build/GroceriStar/sd/tests/favorites.test.js:5:9: 'ingredients' is assigned a value but never used.
error Command failed with exit code 1.
```
I think it's self explanatory

**Comment by atherdon** [comment]
tell me what exact is not clear. and we're pretty close to kills this thing - don't give up

**Comment by AndrewSerra** [comment]
Oh, that's just an unused import all handled. Trying again

**Pull request merged**